packages/dashboard/src/components/dragLayer/itemTypes.ts

```diff
@@ -2,4 +2,5 @@ export enum ItemTypes {
   Grid = 'Grid',
   Component = 'Component',
   ResourceExplorerAssetProperty = 'ResourceExplorerAssetProperty',
+  ResourceExplorerAlarm = 'ResourceExplorerAlarm',
```

Comment:
is this being used? I don't see any other reference to it

Reply:
It's part of the scaffolding we'll need to drop alarms. I could remove it, but we'll just immediately add it back in.